generator/internal/mixin_utils_test.cc

```diff
@@ -219,28 +222,39 @@ TEST_F(MixinUtilsTest, GetMixinMethods) {
             "google.cloud.location.Locations.GetLocation");
   EXPECT_EQ(get_location.grpc_stub_name, "locations_stub");
   EXPECT_EQ(get_location.grpc_stub_fqn, "google::cloud::location::Locations");
```

Comment:
It can be annoying to write test expectations for every field in a proto. Consider using `google::protobuf::TextFormat::ParseFromString()` + `google::cloud::testing_util::IsProtoEqual()`. e.g.

https://github.com/googleapis/google-cloud-cpp/blob/39a3faf28d22440aedd371d21bdf158e847f0aca/google/cloud/storage/internal/grpc/bucket_metadata_parser_test.cc#L307-L327

Reply:
I change that to use `ElementsAre(value1_matcher, value2_matcher, value3_matcher)` in the end.